src/app/page.tsx

```diff
@@ -58,6 +59,7 @@ export default function Index() {
           </div>
         ))}
       </div>
+      <InkeepChatButton />
```

Comment:
Is it intentional to have the button only on the main page and not on any of the use case pages?

Reply:
Nope, this was wrong. Thanks for noticing! Moved to the right place now.